CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add a new section "Known Browser Issues" in FAQ.html.
 - Refactor some types to avoid a circular dependency (#1565).
 - Update package.json to include npm 8.
+- No-op `enableUnifiedPlanForChromiumBasedBrowsers` in `MeetingSesstionConfiguration` (always regarded as true), as Plan B SDP will be fully deprecated from Chromium soon.
```

Comment:
Maybe suffix `This will have no effect on SDK behavior` and has been the default since X.X`.  Also just link the deprecation plan.

```
- Ignore `enableUnifiedPlanForChromiumBasedBrowsers` value (i.e. treat as always equaling the current default value of `true`) in `MeetingSesstionConfiguration`.  Chrome is in the processing of deprecating and removing Plan-B which would cause breakage in applications still trying to use it.  This will have no effect on SDK behavior` and has been the default since X.X.
```